boards/pic32-wifire/Makefile.features

```diff
@@ -0,0 +1,9 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_timer
+FEATURES_PROVIDED += periph_uart
```

Comment:
I tweaked the comment  when squashing see https://github.com/RIOT-OS/RIOT/pull/6066/commits/e2d7d3617ec1e7753d0a3e2406eebdf285d4563d